core-blocks/latest-comments/edit.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import {
+	TextControl,
+	ToggleControl,
+	ServerSideRender,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import {
+	InspectorControls,
+	BlockAlignmentToolbar,
+	BlockControls,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies.
+ */
+import './editor.scss';
+
+const MIN_COMMENTS = 1;
+const MAX_COMMENTS = 100;
+
+class latestComments extends Component {
+	constructor() {
+		super( ...arguments );
+		this.toggleHandler = this.toggleHandler.bind( this );
+		this.changeCommentsToShow = this.changeCommentsToShow.bind( this );
+	}
+
+	toggleHandler( propName ) {
+		return () => {
+			const value = this.props.attributes[ propName ];
+			const { setAttributes } = this.props;
+
+			setAttributes( { [ propName ]: ! value } );
+		};
+	}
+
+	changeCommentsToShow( commentsToShow ) {
+		const { setAttributes } = this.props;
+
+		setAttributes( { commentsToShow: parseInt( commentsToShow, 10 ) || 0 } );
+	}
+
+	render() {
+		const { setAttributes } = this.props;
+		const { align, displayAvatar, displayTimestamp } = this.props.attributes;
+
+		return (
+			<Fragment>
+				<BlockControls key="controls">
+					<BlockAlignmentToolbar
+						value={ align }
+						onChange={ ( nextAlign ) => {
+							setAttributes( { align: nextAlign } );
+						} }
+						controls={ [ 'left', 'center', 'right', 'wide', 'full' ] }
+					/>
+				</BlockControls>
+				<InspectorControls key="inspector">
+					<h3>{ __( 'Latest Comments Settings' ) }</h3>
```

Comment:
`<PanelBody title={ __( 'Latest Comments Settings' ) }>` like in latest-posts looks a little bit nicer.